efficientdet/keras/utils_keras.py

```diff
@@ -30,15 +49,9 @@ def batch_normalization(is_training_bn: bool,
   else:
     gamma_initializer = tf.ones_initializer()
 
-  if data_format == 'channels_first':
-    axis = 1
-  else:
-    axis = -1
-  if strategy in ['tpu', 'gpu'] and is_training_bn:
-    batch_norm_class = tf.keras.layers.experimental.SyncBatchNormalization
```

Comment:
I haven't verified tf.keras.layers.experimental.SyncBatchNormalization can work well for TPU or horovod, so maybe we can just use the existing bn for now?